test/expected/append-14.out

```diff
@@ -2260,6 +2266,123 @@ ANALYZE i3030;
 
 DROP TABLE i3030;
 RESET enable_seqscan;
+--parent runtime exclusion tests:
+--optimization works with ANY (array)
+:PREFIX
+SELECT *
+FROM append_test a
+WHERE a.attr @> ANY((SELECT coalesce(array_agg(attr), array[]::jsonb[]) FROM join_test_plain WHERE temp > 100)::jsonb[]);
+                             QUERY PLAN                             
+--------------------------------------------------------------------
+ Custom Scan (ChunkAppend) on append_test a (actual rows=0 loops=1)
+   Runtime excluded entire hypertable: true
```

Comment:
This should be a counter since it is a per loop decission. Also not a fan of the boolean in EXPLAIN output i would prefer two distinct messages for runtime exclusion variant. Exclusion information should only be visible with EXPLAIN (verbose).
eg:
Chunks excluded during runtime: 123
and
Hypertables excluded during runtime: 4